wagtail_extensions/blocks.py

```diff
@@ -37,3 +38,45 @@ def get_context(self, value, parent_context=None):
         ctx = super().get_context(value, parent_context=parent_context)
         ctx['has_url'] = 'url' in value
         return ctx
+
+
+class AddressBlock(blocks.StructBlock):
+
+    lines = blocks.ListBlock(blocks.CharBlock(label="Line", required=False))
+
+
+class PhoneBlock(blocks.FieldBlock):
+
+    def __init__(self, required=True, help_text=None, **kwargs):
+        self.field = PhoneNumberField(required=required, help_text=help_text)
+        super().__init__(**kwargs)
+
+    def get_prep_value(self, value):
+        return str(value)
+
+
+class DepartmentBlock(blocks.StructBlock):
+
+    name = blocks.CharBlock(required=False)
+    phone = PhoneBlock(required=False)
```

Comment:
We always end up hacing to put more than one phone number in, so I think this needs to be a list block.